aws/table_aws_securityhub_standards_subscription.go

```diff
@@ -0,0 +1,181 @@
+			},
+			{
+				Name:        "enabled_by_default",
+				Description: "Indicates whether the standard is enabled by default.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "standards_status",
+				Description: "The status of the standard subscription.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     GetEnabledStandards,
+			},
+			{
+				Name:        "standards_subscription_arn",
+				Description: "The ARN of a resource that represents your subscription to a supported standard.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     GetEnabledStandards,
+			},
+			// JSON columns
+			{
+				Name:        "standards_input",
+				Description: "A key-value pair of input for the standard.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     GetEnabledStandards,
+			},
+
+			// Standard columns for all tables
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("StandardsArn").Transform(arnToAkas),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listSecurityHubStandardSubcriptions(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	plugin.Logger(ctx).Trace("listSecurityHubStandardSubcriptions", "AWS_REGION", region)
+
+	// Create session
+	svc, err := SecurityHubService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// List call
+	resp, err := svc.DescribeStandards(&securityhub.DescribeStandardsInput{})
+	if err != nil {
+		plugin.Logger(ctx).Error("listSecurityHubStandardSubcriptions", "query_error", err)
+		return nil, nil
+	}
+	for _, standards := range resp.Standards {
+		d.StreamListItem(ctx, standards)
+	}
+	return nil, nil
+}
+
+//// HYDRATE FUNCTIONS
+
+func getSecurityHubStandardSubcription(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getSecurityHubStandardSubcription")
+
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	name := d.KeyColumnQuals["name"].GetStringValue()
+
+	// get service
+	svc, err := SecurityHubService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// Build the input
+	input := &securityhub.DescribeStandardsInput{}
+
+	// Get call
+	standards, err := svc.DescribeStandards(input)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, item := range standards.Standards {
+		if *item.Name == name {
```

Comment:
This is not a get call , here we are just filtering list call.
Do we need this function in this case?